.ci/deploy.sh

```diff
@@ -9,15 +9,15 @@ function doCompile {
 # Pull requests and commits to other branches shouldn't try to deploy, just build to verify
 if [[ ! -d $OUTPUT_PATH ]]; then
   mkdir "$OUTPUT_PATH"
-  echo "Skipping deploy; just doing a build."
+  echo "::warning::Not running from master... doing a build, but skipping deploy!"
```

Comment:
While this is interesting information it's not important enough to warn about.

Reply:
The idea was to extract that useful information and bring it to the log viewers attention.
I found no other way to highlight stuff in the log. The cool part is that it's automatically displayed in the overview page of each workflow:
![Untitled](https://user-images.githubusercontent.com/9874850/101019891-ee42e180-356d-11eb-9a8c-fa287bace187.png)

There is no similar option like `::hint::`, `::highlight::`, `::annotation::`, `::information::` etc. I'm aware of.
I know, I'm slightly misusing warning here... :)

Should I remove that?

Reply:
This isn't a warning, and will lead to confusion down the line imo. If they add annotations in the future, we can add those. Until then, I'm fine silently moving on.

Reply:
Removed the warning tag.